tests/misra/test_mutation.py

```diff
@@ -13,27 +13,50 @@
 # - at least one violation in each safety/safety*.h file
 # - come up with a pattern for each rule (cppcheck tests probably have good ones?)
 mutations = [
+  # default
   (None, None, False),
-  # F4 only
-  ("board/stm32fx/llbxcan.h", "s/1U/1/g", True),
-  # H7 only
-  ("board/stm32h7/llfdcan.h", "s/return ret;/if (true) { return ret; } else { return false; }/g", True),
-  # general safety
-  ("board/safety/safety_toyota.h", "s/is_lkas_msg =.*;/is_lkas_msg = addr == 1 || addr == 2;/g", True),
```

Comment:
let's keep these (I won't count it against the 2m). it's very easy to see that we cover at least those three cases by having this